unit-tests/live/streaming/test-y16.cpp

```diff
@@ -0,0 +1,52 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+//#test:device D400*
+
+#include "../../catch.h"
+#include "../../unit-tests-common.h"
+
+using namespace rs2;
+
+rs2::stream_profile get_ir_y16_profile(rs2::depth_sensor depth_sensor)
+{
+    auto depth_profiles = depth_sensor.get_stream_profiles();
+    rs2::stream_profile ir_profile;
+    for (auto& p : depth_profiles)
+    {
+        if (p.format() == RS2_FORMAT_Y16)
+        {
+            ir_profile = p;
+            break;
+        }
+    }
+    return ir_profile;
+}
+
+// Y16 CONFIGURATION TESTS
+TEST_CASE("Y16 streaming", "[live]") 
+{
+    rs2::context ctx;
+    auto list = ctx.query_devices();
+    REQUIRE(list.size());
+    auto dev = list.front();
+    
+    auto depth_sensor = dev.query_sensors()[0];
+    auto y16_profile = get_ir_y16_profile(depth_sensor);
+    REQUIRE(y16_profile);
+    depth_sensor.open(y16_profile);
+    bool y16_streamed = false;
+    depth_sensor.start([&y16_streamed](rs2::frame f){
+        y16_streamed = true;
+    });
+    while(!y16_streamed)
+    {
+        std::this_thread::sleep_for(std::chrono::milliseconds(5));
+    }
+
+    // if got here, y16 has been streamed
+    REQUIRE(true);
```

Comment:
If using `timer` it can be replaced with
```
REQUIRE(!t.has_expired()); // fail on timeout
```

Reply:
done